lib/request.js

```diff
@@ -103,7 +103,8 @@ const ajax = ({ method, url, headers, body = null }, callback) => {
   xhr.onload = () => {
     $html.classList.remove('swap-is-loading');
     if (xhr.status !== 200) {
-      alert('Error: ' + xhr.status);
+      document.write(xhr.responseText);
+      location.href = xhr.responseURL;
```

Comment:
@joeosburn pretty sure these two lines won't produce what you'd expect. location.href = foo is going to cause a hard refresh but the other line is going to set the page's text.

Reply:
👍 Check out the update I just did